ikarus/finiteelements/mechanics/materials/muesli/mueslimaterials.hh

```diff
@@ -0,0 +1,126 @@
+ */
+template <typename MPT>
+auto makeLinearElasticity(const MPT& mpt) {
+  auto muesliParameters = propertiesFromIkarusMaterialParameters(mpt);
+  return SmallStrain<Muesli::LinearElasticity>(muesliParameters);
+}
+
+/**
+ * \brief Constructs a muesli NeoHooke material model based on muesli::neohookeanMaterial
+ *
+ * \tparam MPT the type of the Ikarus material parameters
+ * \param mpt the Ikarus material parameters
+ * \param useDeviatoricStretches tells the material to use deviatopric principal stretches
+ * \return auto the constructed material
+ */
+template <typename MPT>
+auto makeNeoHooke(const MPT& mpt, bool useDeviatoricStretches = false) {
+  auto muesliParameters = propertiesFromIkarusMaterialParameters(mpt);
+  if (useDeviatoricStretches)
+    addRegularizedTag(muesliParameters);
+  return FiniteStrain<Muesli::NeoHooke>(muesliParameters);
+}
+
+/**
+ * \brief Constructs a muesli StVenantKirchhoff material model based on muesli::svkMaterial
+ *
+ * \tparam MPT the type of the Ikarus material parameters
+ * \param mpt the Ikarus material parameters
+ * \return auto the constructed material
+ */
+template <typename MPT>
+auto makeSVK(const MPT& mpt) {
+  auto muesliParameters = propertiesFromIkarusMaterialParameters(mpt);
+  return FiniteStrain<Muesli::StVenantKirchhoff>(muesliParameters);
+}
+
+/**
+ * \brief Constructs a muesli ArrudaBoyce material model based on muesli::arrudaboyceMaterial
+ *
+ * \param C1 C1 parameter
+ * \param lambda_m lambda_m parameter
+ * \param K the bulk modulus
+ * \param compressible tells the material to use the compressible version of ArrudaBoyce
+ * \return auto the constructed material
+ */
+inline auto makeArrudaBoyce(double C1, double lambda_m, double K, bool compressible = true) {
+  auto muesliParameters = muesli::materialProperties{};
+  muesliParameters.insert({"c1", C1});
+  muesliParameters.insert({"lambdam", lambda_m});
+  muesliParameters.insert({"bulk", K});
+  if (compressible)
+    addCompressibleTag(muesliParameters);
+  return FiniteStrain<Muesli::ArrudaBoyce>(muesliParameters);
+}
+
+/**
+ * \brief Constructs a muesli Yeoh material model based on muesli::yeohMaterial
+ *
+ * \param C c1, c2, c3 parameters in a std::array
+ * \param K the bzlk modolus
+ * \param compressible tells the material to use the compressible version of Yeoh
+ * \return auto the constructed material
+ */
+inline auto makeYeoh(std::array<double, 3> C, double K, bool compressible = true) {
+  auto muesliParameters = muesli::materialProperties{};
+  muesliParameters.insert({"c1", C[0]});
+  muesliParameters.insert({"c2", C[1]});
+  muesliParameters.insert({"c3", C[2]});
+  muesliParameters.insert({"bulk", K});
+  if (compressible)
+    addCompressibleTag(muesliParameters);
+  return FiniteStrain<Muesli::Yeoh>(muesliParameters);
+}
+
+/**
+ * \brief Constructs a muesli MooneyRivlin material model based on muesli::mooneyMaterial
+ *
+ * \param alpha alpha0, alpha1, alpha2 parameters in a std::array
```

Comment:
Ah I didn't knew this, Thanks

Reply:
Should I rename all the variables according to Hyperelastic PR, maybe we can later then synchronise the Parameter structs as well?

Reply:
Yes sure that's even better. But then you can perhaps wait until #333 is merged to avoid unnecessary iterations on the changes and mark this conversation as unresolved.